packages/harmony/src/components/input/SelectablePill/SelectablePill.tsx

```diff
@@ -73,12 +96,34 @@ export const SelectablePill = (props: SelectablePillProps) => {
     }
   }
 
-  return (
-    <SelectablePillRoot {...props}>
+  const pillContent = (
+    <>
       {Icon ? <Icon css={iconCss} /> : null}
       <Text variant='body' tag='span'>
         {label}
       </Text>
-    </SelectablePillRoot>
+    </>
   )
+
+  if (type === 'checkbox' || type === 'radio') {
+    return (
+      <label css={[css, { position: 'relative' }]}>
+        <InputRoot
+          {...other}
+          checked={type === 'checkbox' ? isSelected : undefined}
```

Comment:
do we want `useControlled` here?

Reply:
yes long term for sure. in this case it's always controlled